HHS/OPRE-OPS
# Remove libraries from Pipfile that we don't need at the top level

# Issue 

+ #163

# How to test locally

1. Pull down this branch
2. `docker-compose down && docker-compose build && docker-compose up`
3. Verify that the local Django admin functionality works as expected

# Notes

Some of the libraries that were listed in our `Pipfile` may be sub-dependencies; in that case we don't need to list them explicitly at the `Pipfile` level. `Pipfile.lock` gathers and declares sub-dependencies: https://pipenv.pypa.io/en/latest/basics/#pipenv-lock

Some like `flake8` we intend to use in the future, but let's remove from the `Pipfile` until we've implemented and/or documented how we'll use them.

Some of the libraries listed may just be unused at this point.
